public-api.txt

```diff
@@ -0,0 +1,16 @@
+# This file lists the APIs that are considered "public". It is used by the
+# crd-swagger tool to generate the OpenAPI document for Rancher API reference
+# page.
+#
+# The file is a line-separated list of `<Kind>.<group>`
+Kubeconfig.ext.cattle.io
+Token.ext.cattle.io
+
+AuditPolicy.auditlog.cattle.io
```

Comment:
Since there was some miscommunication I figured I'd verify for this one as well..

@joshmeranda was this meant to be part of the public API where user are expected to configure auditpolicy using kubectl or was it meant to stay internal?

Reply:
OK sorry for the confusion, this is by Frameworks indeed.

@joshmeranda : I am going to reiterate as other elements in this list were picked by mistake, please bear with me if this is already known to you:

We want to be careful as to what we label as "Public API" (RK-API) from now on. Anything that gets in there should remain as backwards-compatible as possible in future releases, with a migration webhook as the last resort. If you and the Team are OK with that, it can stay (cc @prachidamle)